lib/checkout_sdk/common/data_range_query_filter.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module CheckoutSdk
+  module Common
+    # @!attribute from
+    #   @return [DateTime]
+    # @!attribute to
+    #   @return [DateTime]
+    class DataRangeQueryFilter
```

Comment:
I think there is a typo here. It should be `DateRangeQueryFilter` right?

Reply:
Ahh, took me a while to understand this, but yes it is `Date`